### ISSUE TITLE
Fetching community numbers from the API

### ISSUE THREAD

**Issue by apoorv-on-git** [virtual-tech-school/virtual-tech-school#8]
Our home page takes a few seconds to render, technically because we are using APIs and web scraping to fetch 4 numbers -

1. YouTube subscriber count
2. Twitter follower count
3. Discord member count
4. Hashnode follower count

We fetch these values when the page is getting loaded from the views. Instead, we could bring them from the APIs (api.py) and update the numbers by making a get request to the API we create to improve the speed of the page.

**Comment by VenkataBhaskarr** [comment]
yes, I think this is the reason of high initial server response time as mentioned in the page speed insights which is taking almost 3 seconds to load
<img width="965" alt="Screenshot 2023-01-15 at 8 44 02 PM" src="https://user-images.githubusercontent.com/111889155/212549512-3be2cacd-5b73-45f3-9e5d-ae9beda2b93f.png">
